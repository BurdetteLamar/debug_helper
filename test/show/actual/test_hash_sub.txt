---
DebugHelperTest::HashSub (message='test_hash_sub'):
  size: 2
  default: 
  default_proc: 
  Pair 0:
    Key:
      Symbol:
        to_s: a
        size: 1
        encoding: !ruby/encoding US-ASCII
    Value: Fixnum 0
  Pair 1:
    Key:
      Symbol:
        to_s: b
        size: 1
        encoding: !ruby/encoding US-ASCII
    Value: Fixnum 1
