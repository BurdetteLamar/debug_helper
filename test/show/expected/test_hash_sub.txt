---
DebugHelperTest::HashSub (message='test_hash_sub'):
  DebugHelperTest::HashSub#size: 2
  DebugHelperTest::HashSub#default: 
  DebugHelperTest::HashSub#default_proc: 
  Pair 0:
    Key:
      Symbol:
        Symbol#to_s: a
        Symbol#size: 1
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: Fixnum 0
  Pair 1:
    Key:
      Symbol:
        Symbol#to_s: b
        Symbol#size: 1
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: Fixnum 1
